ReactSkia/sdk/OnScreenKeyBoard.cpp

```diff
@@ -0,0 +1,622 @@
+ /*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "include/core/SkFont.h"
+#include "include/core/SkFontMgr.h"
+
+#include "ReactSkia/utils/RnsUtils.h"
+
+#include "OnScreenKeyBoard.h"
+#include "OSKConfig.h"
+#include "OSKLayout.h"
+
+namespace OSK {
```

Comment:
are we defining seperate namespace for OSK ?
shouldnt this be under rns:sdk namespace

Reply:
looks rns:sdk namespace is not used in any of our existing dk's. Are we defining this as new standard for sdk's ?

Reply:
I Discussed, will change namespace OSK to rns:sdk